### ISSUE TITLE
README.md from the Samples/Apps should be moved to the main guide README.md

### ISSUE THREAD

**Issue by xpat** [GoogleCloudPlatform/k8s-config-connector#9]
The main README.md file should incorporate the instructions from the Samples/App README.md file. 

> Prerequisites
> Before trying any of the samples you must run through the following prerequisites to get your environment configured.
> 
> In addition, it is assumed your system has the following.
> 
> cnrm
> gcloud
> kustomize
> Create a GKE cluster with CNRM support.

**Comment by xpat** [comment]
Also, what does CNRM mean?  Running, as instructed, **cnrm init**from the command line yields the following: 

-su: cnrm: command not found

**Comment by AlexBulankou** [comment]
@xpat, thanks for reporting the issue! The only guide required to onboard on Config Connector is the main readme: [/GoogleCloudPlatform/k8s-config-connector/tree/master/README.md](/GoogleCloudPlatform/k8s-config-connector/tree/master/README.md)
The readme under samples/apps ( [/GoogleCloudPlatform/k8s-config-connector/tree/master/samples/apps/README.md](/GoogleCloudPlatform/k8s-config-connector/tree/master/samples/apps/README.md) ) is a left over from our internal guide and should be cleaned up from GitHub.

**Comment by spew** [comment]
@AlexBulankou I think @xpat is suggesting that the main readme needs some additional content that is in the samples/app Readme right now.

**Comment by xpat** [comment]
@spew @AlexBulankou Yes, in particular the instructions to install jq.  That might be the only thing that was missing from the main readme that gave me an error.  The command cnrm init in the samples/app readme is no longer needed, right?  (Still wondering what cnrm stands for or if its just a uniqe, arbitrary namespace that refers to the config-connector, but why isn't it necessary to run cnrm init?)

**Comment by kibbles-n-bytes** [comment]
"cnrm" stands for "Cloud Native Resource Management". Yeah, we used to have a standalone CLI, but we removed it. We'll update that doc to not reference it.

**Comment by xpat** [comment]
Got it, thanks.
https://www.youtube.com/watch?v=s_hiFuRDJSE&feature=youtu.be

- Christophe R. Patraldo
[image: https://www.facebook.com/pinchepoutine]
<https://facebook.com/pinchepoutine>
